tsconfig.eslint.json

```diff
@@ -0,0 +1,20 @@
+{
+  // extend your base config so you don't have to redefine your compilerOptions
```

Comment:
i dont think json supports this type of comment

Reply:
well, it still works.  but I will remove it.

Reply:
done